cpp/dolfinx/mesh/utils.h

```diff
@@ -64,7 +64,10 @@ graph::AdjacencyList<std::int64_t>
 extract_topology(const CellType& cell_type, const fem::ElementDofLayout& layout,
                  const graph::AdjacencyList<std::int64_t>& cells);
 
-/// Compute greatest distance between any two vertices
+/// Compute greatest distance between any two vertices in the entity
+/// @param[in] mesh The mesh
+/// @param[in] entities List of entities (local to process)
```

Comment:
`/// @param[in] entities List of entity indices to compute the size measure for.`